tempesta_fw/str.h

```diff
@@ -92,13 +92,17 @@
  * @skb		- socket buffer containign the string data;
  * @len		- total length of compund or plain string (HTTP message body
  *		  size can be extreme large, so we need 64 bits to handle it);
+ * @eol_len     - the length of string's line endings, if present (as for now,
+ *                it should be 0 if the string has no EOL at all, 1 for LF and
+ *                2 for CRLF);
  * @flags	- 3 most significant bytes for number of chunks of compound
  * 		  string and the least significant byte for flags;
  */
 typedef struct {
 	void		*ptr;
 	struct sk_buff	*skb;
 	unsigned long	len;
+	unsigned char   eol_len;
```

Comment:
Personally, I like explicit type names like `uint8_t`. But as I see it, Tempesta doesn't use such a types. So, what's the benefit of using `uint8_t` here instead of `unsigned char`?